app/src/main/res/xml/preferences.xml

```diff
@@ -229,6 +229,10 @@
             android:icon="@drawable/ic_changelog"
             android:title="@string/show_changelog"
             android:summary="@string/show_changelog_summary" />
+        <SwitchPreference
```

Comment:
Please update the screenshot description 🙏🏻

Reply:
Updated the screenshots in the description.